CatNote.API/Controllers/AchievementController.cs

```diff
@@ -0,0 +1,14 @@
+﻿using CatNote.API.DTO;
+using CatNote.BLL.Interfaces;
+using CatNote.BLL.Mappers.Abstractions;
+using CatNote.BLL.Models;
+
+namespace CatNote.API.Controllers;
+
+public class AchievementController : GenericController<AchievementModel, AchievementDTO>
+{
+    public AchievementController(IMapper<AchievementModel, AchievementDTO> mapper, IGenericService<AchievementModel> service)
+        : base(mapper, service)
+    {
+    }
+}
```

Comment:
You forgot to edd empty lines. Google how to set it by default after saving a file in VisualStudio